content/en/Platform Deep Dive/Collaboration/Organization/user-roles.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "User Roles and Permissions"
```

Comment:
@mjang-cobalt I decided to merge several smaller permissions into one and added more items to the table. I'm still waiting for confirmation on some integrations-related roles.

Reply:
@mjang-cobalt My comment above is no longer relevant. The permissions related to integrations that I listed are correct.
Another bigger question is whether we want to include _all_ permissions. I don't think so. Many permissions in the internal spreadsheet are geared towards our internal Cobalt audience. But I'm open to suggestions.

Reply:
In the future, I want to pull in our (internal) CSV SSoT for these permissions -- and we can with https://gohugo.io/templates/data-templates/. But we don't have time to set that up today.

Reply:
@mjang-cobalt I would do it only if the source spreadsheet is polished and geared towards our users. Moreover, it contains staff-only permissions. If there is a way to filter specific columns, that could be interesting. But now it's easier to create a table manually.

Reply:
> If there is a way to filter specific columns

There are ways:

- https://discourse.gohugo.io/t/csv-display-help/17928/8
- https://randomgeekery.org/post/2020/06/csv-and-data-tables-in-hugo/

I guess the reason I hesitate: I'd rather use our code as a source, rather than a spreadsheet. AFAIK, our developers also use a manual process to "copy" the source spreadsheet.